core/bones/base.py

```diff
@@ -441,19 +440,22 @@ def parseSubfieldsFromClient(self) -> bool:
         """
         return False
 
-    def singleValueFromClient(self, value, skel, name, origData):
-        """
-        Prevents the BaseBone from reading data using the fromClient method.
-        If needed, use the RawBone instead.
-        Derived bones should overwrite this method for proper data processing.
-
-        :param value: The value to be processed.
-        :param skel: The skeleton containing the bone.
-        :param name: The name of the bone.
-        :param origData: The original data from the client.
-
-        :return: A tuple containing the empty value and a list with a ReadFromClientError.
+    def singleValueFromClient(self, value: Any, skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
+        """Load a single value from a client
+
+        :param value: The single value which should be loaded.
+        :param skel: The SkeletonInstance where the value should be loaded into.
+        :param bone_name: The bone name of this bone in the SkeletonInstance.
+        :param client_data: The data we got from the client,
```

Comment:
I hate the term "we" in documentation. The program does something, not "we".
```suggestion
        :param client_data: The data taken from the client,
```